src/index.js

```diff
@@ -29,6 +30,7 @@ io.on('connection', (socket) => {
 
   socket.on('join', (res) => {
     console.log(`someone joined ${res.pathName}`)
+    sendEmail(res.email, res.username, res.pathName);
```

Comment:
please where is `res.email` coming from

Reply:
It's coming from the frontend. I don't want to hardcode it on the backend. Thinking of how we might want to use it in the future. We just have to send the customer service emails from the frontend maybe as props. I don't know if you understand